tools/codegen/api/unboxing.py

```diff
@@ -0,0 +1,192 @@
+from typing import List, Tuple
+
+from tools.codegen.api.cpp import argumenttype_type
```

Comment:
nit: we have a whole bunch of `argumenttype_type` functions that are all under different api's (`dispatcher.argumenttype_type`, `python.argumenttype_type`, etc).

Can you just `from tools.codegen.api import cpp`, that way you have to use `cpp.argumenttype_type` directly and it's clear which api it is?